cli/operator/node.go

```diff
@@ -736,68 +716,57 @@ func setupEventHandling(
 		fromBlock = new(big.Int).SetUint64(fromBlock.Uint64() + 1)
 	}
 
-	// load & parse local events yaml if exists, otherwise sync from contract
-	if len(cfg.LocalEventsPath) != 0 {
-		localEvents, err := localevents.Load(cfg.LocalEventsPath)
-		if err != nil {
-			logger.Fatal("failed to load local events", zap.Error(err))
-		}
+	// Sync historical registry events from Ethereum smart contract.
+	logger.Debug("syncing historical registry events", zap.Uint64("fromBlock", fromBlock.Uint64()))
 
-		if err := eventHandler.HandleLocalEvents(localEvents); err != nil {
-			logger.Fatal("error occurred while running event data handler", zap.Error(err))
-		}
-	} else {
```

Comment:
https://github.com/ssvlabs/ssv/pull/1819/files#r1826980844 logic for on-chain events